pulser-core/pulser/sequence/sequence.py

```diff
@@ -774,6 +779,7 @@ def enable_eom_mode(
         amp_on: Union[float, Parametrized],
         detuning_on: Union[float, Parametrized],
         optimal_detuning_off: Union[float, Parametrized] = 0.0,
+        correct_phase_drift: bool = False,
```

Comment:
Because it will change the default behaviour and thus break backwards compatibility with previously written code. Plus, I always err on the side of letting the user choose the add-ons, even if I believe they will be generally beneficial

Reply:
Okay, make sense then. As long as it clearly documented and recommended somewhere in the docs, it's fine.